src/borg/shellpattern.py

```diff
@@ -15,7 +15,7 @@ def translate(pat, match_end=r"\Z"):
 
     This function is derived from the "fnmatch" module distributed with the Python standard library.
 
-    Copyright (C) 2001-2017 Python Software Foundation. All rights reserved.
+    Copyright (C) 2001-2018 Python Software Foundation. All rights reserved.
```

Comment:
maybe we should not update that one, it was copied in 2017 from the py stdlib.

Reply:
Actually it was copied back in 2016 at #1989, but I modified that line last year for similar reasons. I'm sorry for this. Should we change it back to 2016?

Reply:
yes, maybe change it back.